### PR TITLE
Changing Cmake to bypass only IPv6 tests for Travis

### PR DESCRIPTION
**Pull request by draganglumac**

### CODE REVIEW

File: test/unit/main.c [hunk 1]
```diff
@@ -66,25 +66,14 @@ int main(int argc, char **argv) {
...
-#endif
-#ifdef TEST_DISABLE_IPC
-  JNXLOG(LDEBUG,"Running with disabled IPC tests");
-  tsize--;
-#endif
-#ifdef TEST_DISABLE_UDP
-  JNXLOG(LDEBUG,"Running with disabled UDP tests");
-  tsize--;
+#ifdef TEST_DISABLE_IPV6
+  JNXLOG(LDEBUG,"Running with disabled IPv6 tests");
```
Comment (AlexsJones) [inline test/unit/main.c]: I am suprised this would work if you are using `test_suite_size` without accounting for the removed test

**Comment by draganglumac** [comment]
I am not removing any tests though just not compiling IPv6 tests in TCP and UDP tests using #ifndef

**Comment by AlexsJones** [comment]
I am trying to help diagnose the build failure, i cannot reproduce it locally. I might suggest we comment out the logger and try seeing if its an error in that system

File: src/ipc/jnx_ipc_socket.c [hunk 2]
```diff
@@ -127,12 +127,12 @@ void jnx_socket_ipc_listener_tick(jnx_ipc_listener* listener,
       }while(new_fd != -1);
     }else {
       jnx_size rc = 0;
-      JNXLOG(0,"Descriptor is readable %d",listener->ufds[i].fd);
+      // JNXLOG(LDEBUG,"Descriptor is readable %d",listener->ufds[i].fd);
       close_conn = 0;
       compress_array = 0;
       char buffer[MAXBUFFER];
       bzero(buffer,MAXBUFFER);
-      rc = recv(listener->ufds[i].fd,buffer,sizeof(buffer),0);
```
Comment (AlexsJones) [inline src/ipc/jnx_ipc_socket.c]: Good spot

**Pull request merged**